### PR TITLE
MAINTAINERS: move mesh and ll from bsim/tests to their proper section

### PR DESCRIPTION
**Pull request by kruithofa**
The babblesim tests for bluetooth have been moved to a new folder.
To ensure that the proper people still get involved this PR adds
the bsim/tests/bluetooth/ll folder to the bluetooth controller section
and similar for the mesh babblesim folder
    
Signed-off-by: Andries Kruithof <andries.kruithof@nordicsemi.no>

### CODE REVIEW

File: MAINTAINERS.yml [hunk 1]
```diff
@@ -211,9 +211,11 @@ Bluetooth:
     - subsys/bluetooth/audio/
     - include/zephyr/bluetooth/audio/
     - tests/bsim/bluetooth/audio/
+    - tests/bsim/bluetooth/ll/
```
Comment (cvinayak) [inline MAINTAINERS.yml:214]:
```suggestion
    - tests/bsim/bluetooth/host/
```

**Review by cvinayak — DISMISSED** [review]

File: MAINTAINERS.yml (hunk 1 above)
Comment (aescolar) [reply MAINTAINERS.yml:214]: @cvinayak note that this is the exclude section.

Comment (aescolar) [reply MAINTAINERS.yml:214]: @kruithofa please add a `tests/bsim/bluetooth/` in the files list above

**Review by aescolar — CHANGES_REQUESTED** [review]

File: MAINTAINERS.yml (hunk 1 above)
Comment (cvinayak) [reply MAINTAINERS.yml:214]: > @cvinayak note that this is the exclude section.

Ah... I did not notice that.

**Comment by kruithofa** [comment]
@aescolar is doing some more changes and will include these, so this PR becomes obsolete

**Pull request closed without merging**